src/include/ipc/stream.h

```diff
@@ -92,8 +92,10 @@ struct sof_ipc_stream_params {
 
 	/* for notifying host period has completed - 0 means no period IRQ */
 	uint32_t host_period_bytes;
+	uint16_t period_irq;
 
-	uint32_t reserved[2];
+	uint16_t not_used;
```

Comment:
It will look nicer if we merge them together.

Reply:
Sure I can define reserved as `uint16_t ` and make it `uint32_t reserved[5];` then remove `not_used` competely.